Add info on adding a Fleet Server to a policy (#2462)
elastic/observability-docs: docs/en/ingest-management/agent-policies.asciidoc

Any {agent}s enrolled in the agent policy will begin sending data to the
specified outputs.

[discrete]
[[add-fleet-server-to-policy]]

> == Add a {fleet-server} to a policy

Author: @elastic/obs-docs I could use another set of 👀 here...

The [docs build is failing](https://elasticsearch-ci.elastic.co/job/elastic+observability-docs+pull-request+build-docs/2877/console) on this PR with the error `invalid reference: add-fleet-server-to-policy`, but `add-fleet-server-to-policy` _does_ exist (above). I can't figure out what I'm missing...  🤔

Reviewer: @colleenmcginnis It looks like the link is in a shared region. Have you tried using the external link format to see if that fixes the problem?

If you want to connect multiple agents to a specific on-premises {fleet-server},
you can add that {fleet-server} to a policy.

[role="screenshot"]
image::images/add-fleet-server-to-policy.png[Screen capture showing how to add a {fleet-server} to a policy when creating or updating the policy.]

> Once the policy is saved, the configuration on these agents will be modified

Reviewer: ```suggestion
When the policy is saved, all agents assigned to the policy are configured
```

to use the new {fleet-server} as the controller.

> Make sure that the {agent}s in this policy all have connectivity to the {fleet-server}
> adding the {fleet-server} to the policy. Lack of connectivity will prevent the {agent}

Reviewer: ```suggestion
Make sure that the {agent}s assigned to this policy all have connectivity to the {fleet-server}
that you added. Lack of connectivity will prevent the {agent}
```
I think there are some extra words here?
